packages/cicero-core/src/template.js

```diff
@@ -18,6 +18,8 @@ const Metadata = require('./metadata');
 const Logger = require('@accordproject/concerto-core').Logger;
 const ParserManager = require('@accordproject/markdown-template').ParserManager;
 const crypto = require('crypto');
+const fs = require('fs');
```

Comment:
I would pass the content  of the `p12` file to the function, rather than load it from disk in this class. Loading from file could be done in CLI or other auxiliary code as needed. This is more consistent with what we do with other files.